catalogue/webapp/components/WorkDetailsList/WorkDetailsList.tsx

```diff
@@ -4,15 +4,10 @@ import { FunctionComponent } from 'react';
 import Space from '@weco/common/views/components/styled/Space';
 import WorkDetailsProperty from '../WorkDetailsProperty/WorkDetailsProperty';
 
-const PlainList = styled(Space).attrs({
-  v: {
-    size: 'm',
-    properties: ['margin-bottom'],
-  },
-  className: 'no-margin',
-})`
```

Comment:
haha what was this 👀

Reply:
yeah, this was an instance where it looked like the intention was maybe to set all margin sides to `0` and re-add bottom margin with the `Space` component, but evidently misunderstanding that that wouldn't happen since `no-margin` was `!important`. I chose to keep the status quo and not change the appearance (since nobody had been complaining about it up to this point)